idaes/power_generation/unit_models/boiler_heat_exchanger.py

```diff
@@ -720,6 +699,9 @@ def _make_performance(self):
             units=pyunits.K,
         )
 
+        self.delta_temperature_in = Reference(self.deltaT_1)
```

Comment:
This is something for the IDAES 2.0 discussion, but we really should standardize names across all the HX models (including the names for sides and Ports).